src/Component/src/Symfony/Serializer/State/SerializeProcessor.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\src\Symfony\Serializer\State;
+
+use Sylius\Resource\Context\Context;
+use Sylius\Resource\Context\Option\RequestOption;
+use Sylius\Resource\Metadata\Operation;
+use Sylius\Resource\State\ProcessorInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+final class SerializeProcessor implements ProcessorInterface
+{
+    public function __construct(
+        private ProcessorInterface $decorated,
+        private ?SerializerInterface $serializer,
+    ) {
+    }
+
+    /**
+     * Serializes the data to the requested format.
```

Comment:
I think comment can be safely removed. :slightly_smiling_face:  Or moved at class level.

Reply:
done